docs/installation-and-operation/serialization.md

```diff
@@ -27,13 +27,14 @@ There are two ways to run these `export` and `import` commands:
 
 - **Staging environments**. Enable a staging-to-production workflow for important dashboards by exporting from a staging instance of Metabase and then importing them into your production instance(s).
 - **Version control**. Check the exported files into version control and audit changes to them, as the YAML files contained within the export are pretty readable.
+- **Duplicating assets between Metabase instances**. Export the "template" data from a source Metabase instance and import them to one or more target instances to duplicate.
```

Comment:
This seems like a restatement of the first bullet, but I'm not opposed to including it if it helps people get the point.

And should it be "between" or "to"? Does "between" imply bi-directionality?

```suggestion
- **Duplicating assets to other Metabase instances**. Export the "template" data from a source Metabase and import them to one or more target instances.
```